stable/insights-agent/templates/helm-hook/job.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1
```

Comment:
Can we actually gate this job on a variable like `runJobsImmediately`? That might come in handy down the line.

Reply:
Done